services/terraform/bind/main.tf

```diff
@@ -37,6 +37,13 @@ resource "aws_iam_user_policy" "user_policy" {
             "aws:SourceIp" = var.source_ips
           }
         }
+      },
+      {
+        Action: [
+          "logs:FilterLogEvents"
+        ],
+        Effect: "Allow",
+        Resource: "*"
```

Comment:
We need to set the source IP on this action as well. I think since this action is `Effect: Allow` and `Resource: "*"` the easiest way would be to just add `logs:FilterLogEvents` into the existing Action array